### PR TITLE
Add support to inline the plugin with kn binary

### PR DESCRIPTION
**Pull request by dsimansk**
## Description

Add support to inline the plugin with kn binary.

## Changes

* Add `root` pkg to expose `NewSourceKafkaCommand`
* Add `plugin` package for inlining
* Update vendor dir in separated commit

## Reference

Fixes N/A

### CODE REVIEW

**Review by navidshaikh — APPROVED** [review]
/ok-to-test

File: plugin/plugin.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	defer (func() {
+		os.Args = oldArgs
+	})()
+	os.Args = append([]string{"kn-source-kafka"}, args...)
+	return cmd.Execute()
+}
+
+// Description is displayed in kn's plugin section
+func (l *sourceKafkaPlugin) Description() (string, error) {
+	return "Manage Kafka event source", nil
```
Comment (rhuss) [inline plugin/plugin.go:49]:
```suggestion
	return "Manage Kafka sources", nil
```

that aligns with our other wording for the built-in sources.

**Review by rhuss — CHANGES_REQUESTED** [review]
Looks good with a minor wording suggestion

**Review by rhuss — COMMENTED** [review]
/lgtm
/approve

Thanks !

**Pull request merged**